src/path.h

```diff
@@ -111,9 +111,11 @@ enum PathType {
 
 /** Path status. */
 enum PathStatus {
-	PAS_UNUSED,      ///< %Path is not loaded.
-	PAS_NORMAL_PATH, ///< %Path to walk on.
-	PAS_QUEUE_PATH,  ///< %Path to queue on.
+	PAS_NORMAL_PATH,  ///< Path to walk on.
```

Comment:
I think the `%` is about not linking the name to the `Path` class, you'd have to check the doxygen output to check.

Reply:
Shuffling path values doesn't break save games?